checkov/terraform/checks/resource/oci/SecurityListIngressStateless.py

```diff
@@ -0,0 +1,27 @@
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class SecurityListIngressStateless(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure VCN inbound security lists are stateless"
+        id = "CKV_OCI_17"
+        supported_resources = ['oci_core_security_list']
+        categories = [CheckCategories.GENERAL_SECURITY]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        if 'ingress_security_rules' in conf.keys():
+            rules = conf.get("ingress_security_rules")
+            for rule in rules:
+                if 'stateless' in rule.keys():
+                    if rule.get("stateless") == [True]:
+                        return CheckResult.PASSED
+                    else:
+                        return CheckResult.FAILED
+            return CheckResult.PASSED
+
+        return CheckResult.UNKNOWN
+
```

Comment:
@JamesWoolfenden Please set the evaluated keys with the correct index, like we do here - https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/aws/RedShiftSSL.py

Reply:
what do they do?

Reply:
The report will display the key where the violation occurred, and it also helps us create a fix for this check.